src/sections/dataset/dataset-citation/DatasetCitation.tsx

```diff
@@ -0,0 +1,69 @@
+import { Col, Icon, Row, Tooltip } from 'dataverse-design-system'
+import styles from './DatasetCitation.module.scss'
+import { useTranslation } from 'react-i18next'
+import { Citation, DatasetStatus } from '../../../dataset/domain/models/Dataset'
+
+interface DatasetCitationProps {
+  citation: Citation
+  status: DatasetStatus
+  version: string | null
+}
+interface CitationDatasetStatusProps {
+  status: DatasetStatus
+}
+
+function CitationDescription({ citation, status, version }: DatasetCitationProps) {
+  return (
+    <span>
+      {citation.citationText}, {citation.pidUrl}, {citation.publisher}
+      {version && ', V' + version.substring(0, 1)}
+      {citation.unf && ', ' + citation.unf}
+      {status !== DatasetStatus.PUBLISHED && <CitationDatasetStatus status={status} />}
```

Comment:
We could move this logic `status !== DatasetStatus.PUBLISHED &&` inside the CitationDatasetStatus component, because if we'd use CitationDatasetStatus elsewhere, we would need this logic
```suggestion
      <CitationDatasetStatus status={status} />
```